apps/spruce/src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -108,7 +108,7 @@ export const TaskDurationTable: React.FC<Props> = ({
             // @ts-expect-error: FIXME. This comment was added by an automated script.
```

Comment:
the type errors can probably be fixed by typing the columns with `LGColumnDef<type_here>`